ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -15,19 +15,12 @@ group_repo_post() {
 }
 
 distro_custom() {
-    # install avocado
-    local avocado_rpms=(python3-avocado{,-plugins-{output-html,varianter-yaml-to-mux}})
-    if [ -z "$(dnf repoquery "${avocado_rpms[@]}")" ]; then
-        avocado_rpms=()
-        pip install "avocado-framework<83.0"
-        pip install "avocado-framework-plugin-result-html<83.0"
-        pip install "avocado-framework-plugin-varianter-yaml-to-mux<83.0"
-    fi
-    dnf -y install "${avocado_rpms[@]}" clustershell
-
-    # for Launchable's pip install
-    dnf -y install python3-setuptools.noarch
+    # TODO: This code is not exiting on failure.
```

Comment:
@brianjmurrell any thought on this, I thought this script was running "set -e" but failures in this function are ignored which surprised me.

Reply:
Which specific failure is being ignored?